massgov/mayflower-patternlab
# DP-7500 / DP-7600: Updates for Elected Officials

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

(DP-7500) Update the image link list to still use appropriate classes when the links are not passed in (when the block is overridden).
(DP-7600) Update the items used in the about section of the elected official page and extract sub-patterns into separate templates for simpler mapping.


## Related Issue / Ticket

- [DP-7500](https://jira.state.ma.us/browse/DP-7500)
- [DP-7600](https://jira.state.ma.us/browse/DP-7600)

## Steps to Test


*Note: This change does not affect markup of templates or pages, only how that is built or overridden. Because Visual Regression Testing passes without any updates to the Visual Regression Testing images, external review seems minimally necessary.*
1. Checkout the branch: feature/DP-7600--elected-abouts-updates.
1. Build mayflower locally.
1. Confirm that the following patterns haven't changed / match https://mayflower.digital.mass.gov: 
    * Pages->Organization Detail
    * Pages->Organization Elected Official

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?
Because extracting the icon list item into its own pattern causes it to lose formatting, I added a todo indicating that.  It still looks fine within the icon links molecule.

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
